sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py

```diff
@@ -46,7 +46,7 @@ class CertificateClient(KeyVaultClientBase):
         :mod:`azure.identity`
     :keyword api_version: version of the Key Vault API to use. Defaults to the most recent.
     :paramtype api_version: ~azure.keyvault.certificates.ApiVersion
-    :keyword transport: transport to use. Defaults to :class:`~azure.core.pipeline.transport.RequestsTransport`.
+    :keyword transport: transport to use.
```

Comment:
Looking at other client docstrings in the repo, we're actually the only ones that are including a `transport` keyword argument (probably because we were the only ones to incorrectly provide our own default 😅). I think we can remove the `transport` from our client docstring since this is universally configurable and documented in `azure-core`.

Reply:
@mccoyp I'm assuming this applies for the other SDKs too?